src/common/HaskellDo/View.hs

```diff
@@ -67,6 +67,7 @@ widgets state = do
     **> openProjectButtonWidget
     **> packageEditorButtonWidget
     **> toggleEditorButtonWidget
+    **> toggleErrorButtonWidget
     **> compileButtonWidget
     **> pathInputWidget
     **> closeModalButtonWidget
```

Comment:
The code looks great overall, I'm feeling a bit unsafe about the widget threading over here, but it should be fixed with #109 😁